playbooks/roles/learner-profile-dashboard/templates/lpd-pre-backup.sh

```diff
@@ -6,7 +6,9 @@ service gunicorn restart
 
 timestamp=$(date +%Y%m%dT%H:%M:%S)
 
-mv /var/log/lpd/debug.log-* "{{ LPD_LOG_DOWNLOAD_LOG_DIR }}"
+if [ -f /var/log/lpd/debug.log-* ]; then
+   mv /var/log/lpd/debug.log-* "{{ LPD_LOG_DOWNLOAD_LOG_DIR }}"
```

Comment:
@itsjeyd Here leaving `mv` without the `if` clause makes `mv` fail (and consequently, the whole script) when `/var/log/lpd/debug.log-*` doesn't match any file.